src/FakeItEasy/Core/FakeManager.ObjectMemberRule.cs

```diff
@@ -87,7 +87,7 @@ private bool TryHandleToString(IInterceptedFakeObjectCall fakeObjectCall)
                     return false;
                 }
 
-                fakeObjectCall.SetReturnValue("Faked {0}".FormatInvariant(this.fakeManager.FakeObjectType.ToString()));
+                fakeObjectCall.SetReturnValue("Faked " + this.fakeManager.FakeObjectType);

```

Comment:
I think it's fine to leave the `+` operator here. It's optimised anyway, and it does sometimes read better than the equivalent bling string.